cel/program.go

```diff
@@ -283,3 +292,13 @@ func (gen *progGen) Eval(input interface{}) (ref.Val, *EvalDetails, error) {
 	}
 	return v, det, nil
 }
+
+// Cost implements the Program interface method.
+func (gen *progGen) Cost() (min, max int64) {
+	state := interpreter.NewEvalState()
```

Comment:
Consider using a private empty state value for computing cost since no evaluation is performed.

Reply:
like this?